### PR TITLE
C14 headers

### PR DESCRIPTION
**Pull request by jrosen48**
This addresses #194. @emilybovee, can you review the changes I made? I mostly added headers, but also made a few other additions. 
Chapter is great! Thanks for doing such a great job on this - this was an easy series of edits to make.

### CODE REVIEW

File: 14-wt-machine-learning.Rmd [hunk 1]
```diff
@@ -71,6 +85,8 @@ The outcome was the final course grade that the student earned. The predictor va
 
 To interpret our findings, we will  three main factors: (1) predictive accuracy of the random forest model, (2) variable importance, and (3) variance explained by the final random forest model. In this walkthrough, we will the R package **caret** to carry out the analyses.
 
+## Load Packages
```
Comment (emilybovee) [inline 14-wt-machine-learning.Rmd]: Was this intentional? ##Load packages and then ##Analysis? It looks like they are back to back

File: 14-wt-machine-learning.Rmd [hunk 2]
```diff
@@ -440,4 +464,8 @@ defaultSummary(as.data.frame(data_train_randomfor))
...
+
+## Conclusion
+
+In this chapter, we introduced both general machine learning ideas, like a training and test data set and evaluating the importance of specific variables, and specific ideas, like how a random forest works and how to tune specific parameters so that the model is as effective as possible at predicting an outcome. Like many of the topics in this book - but, perhaps *particularly* so for machine learning - there is much more to learn on the topic, and we encourage you to consult the books and resources in the [learning more chapter](#c17) to learn about further applications of machine learning methods
```
Comment (emilybovee) [inline 14-wt-machine-learning.Rmd]: Love this conclusion, it really ties it all together. Well done. Maybe missing a period at the end!

File: 14-wt-machine-learning.Rmd (hunk 1 above)
Comment (jrosen48) [reply 14-wt-machine-learning.Rmd]: Addressed!

File: 14-wt-machine-learning.Rmd (hunk 2 above)
Comment (jrosen48) [reply 14-wt-machine-learning.Rmd]: Thank you! Added.

**Comment by jrosen48** [comment]
and merging! Thanks much @emilybovee

**Pull request merged**